mock-rollout/src/pages/dashboard/Dashboard.tsx

```diff
@@ -1,14 +1,29 @@
 import React, { useContext } from 'react';
-import { HardwareList } from '../../components/Hardware/HardwareList/HardwareList';
-import { HardwareContext } from '../../store/HardwareContext';
+import { ProjectContext } from '../../store/ProjectContext';
+import { ProjectItem } from '../../components/Project/ProjectItem/ProjectItem'; 
+import { Spinner } from '../../components/UI/Spinner/Spinner';
+import { ErrorMessage } from '../../components/UI/ErrorMessage/ErrorMessage';
 import '../dashboard/Dashboard.scss';
 
 export const Dashboard = () => {
```

Comment:
Dashboard page no longer makes sense if you rename the ProjectItem to ProjectPage and you move it here. If you do that, you'll need to use the context in ProjectPage.

If you want to keep the ProjectItem component is fine, but then the Dashboard should render both the ProjectItem and the HardwareList coresponding to that project and keep them in sync with the selected data.

Otherwise is strange for a ProjectItem to render a hardware list, considering that you are referring to an item. Normally I would only look in the ProjectItem.

You can choose between these two implementations: either rename to ProjectPage and use the context in there, either keep the Dashboard and make it a joint between ProjectItem and HardwareList via lifting state up from the components.